README.md

```
@@ -168,6 +177,325 @@ tests/
+@layer components {
+  .my-custom-class {
+    @apply bg-nord-1 text-nord-6 p-4 rounded-lg;
+  }
+}
+```
+
+#### 3. Tailwind IntelliSense
+VS Code를 사용한다면 "Tailwind CSS IntelliSense" 확장 프로그램을 설치하여 자동완성을 활용하세요.
+
+#### 4. DaisyUI 컴포넌트 탐색
+더 많은 DaisyUI 컴포넌트는 공식 문서를 참고하세요: https://daisyui.com/components/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)